cli.mjs

```diff
@@ -77,15 +77,15 @@ async function downloadDataFirebase() {
 
       // Save the session to a unique JSON file. (":" are replaced to prevent issues with invalid file names)
       const outputFile =
-        `${OUTPUT_ROOT}/participant_responses/` +
+        `${OUTPUT_ROOT}/${RESPONSES_COL}/` +
         `${STUDY_ID}/${PARTICIPANT_ID}/${experimentID}.json`.replaceAll(":", "_");
 
       // TODO 172: Check for overwriting file?
       try {
         fsExtra.outputJSONSync(outputFile, experimentData, { spaces: 2 });
-        console.log("Data saved successfully: ", outputFile);
+        console.log("Data saved successfully:", outputFile);
       } catch (error) {
-        console.error("There was an error saving ", outputFile);
+        console.error("There was an error saving", outputFile);

```

Comment:
Why not use template strings? I'm genuinely curious

Reply:
Think I'm just in the habit of not using them when the variable is at the end of the string. Will change!

Reply:
Adding a var as an argument sometimes prints objects better. The template strings coerce to string (which gives you a lot of `[Object object]`). If your var is a string use the template strings, if your var is something more complex it might make sense to keep this as is!

Reply:
I also tend to not use template strings, mostly because it saves some awkward keystrokes.

Reply:
Looks like they're strings so I'll update it!